d2g.go

```diff
@@ -212,6 +212,8 @@ func setFeatures(dwPayload *dockerworker.DockerWorkerPayload, gwPayload *generic
 	gwPayload.Features.ChainOfTrust = dwPayload.Features.ChainOfTrust
 	gwPayload.Features.TaskclusterProxy = dwPayload.Features.TaskclusterProxy
 	gwPayload.Features.Interactive = dwPayload.Features.Interactive
+	gwPayload.Features.LiveLog = dwPayload.Features.LocalLiveLog
+	gwPayload.Features.BackingLog = dwPayload.Features.BulkLog
```

Comment:
Are you sure these two features map one-to-one? I was under the impression that we'd need a couple new features for LocalLiveLog and BulkLog.